common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -50,11 +50,13 @@ $asterisk-icon: '\f069'; // .fa-asterisk
 // ====================
 @mixin status-icon($color: $gray, $fontAwesomeIcon: "\f00d"){
 
-  &:after {
-    @extend %use-font-awesome;
-    color: $color;
-    font-size: 1.2em;
-    content: $fontAwesomeIcon;
+  .status-icon {
```

Comment:
when you are done, I can clean this part up. Now that the color comes from the span class status, we don't need the color in @mixin status-icon($color ...) anymore